pages/0_cadeaubon.php

```diff
@@ -0,0 +1,232 @@
+<?php 
+
+//Set title
+setTitle("Cadeaubon");
+
+//Function for getting the right ammount of pressing on the add button and getting the right value of this pressed button.
+function str_start_with($value, $start){
+    return substr($value, 0, strlen($start)) === $start;
+}
+
+
+
+//Function for adding items to a list of giftcart items.
+function addCard($giftCardValue) {
+    if (!isset($_SESSION['giftcards'][$giftCardValue])) {
+        $_SESSION['giftcards'][$giftCardValue] = 1;
+    } else {
+        $_SESSION['giftcards'][$giftCardValue]++;
+    }
+}
+
+//Function for remove items from the list of choosen giftcart items.
+function removeCard($giftCardValue) {
+    if(!isset($_SESSION['giftcards'][$giftCardValue])){
+        return;
+    }
+    if ($_SESSION['giftcards'][$giftCardValue] <= 1) {
+        unset($_SESSION['giftcards'][$giftCardValue]);
+    } else {
+        $_SESSION['giftcards'][$giftCardValue]--;
+    }
+}   
+
+//Function for errors.
+$errors = [];
+
+function getFilledInDataErrors()
+{
+    $errors = [];
+    
+    if(empty($_POST['Email'])){
```

Comment:
Ik zou ook controleren of de email correct is met behulp van de `is_email_valid($email)` functie.